src/sentry/seer/code_review/webhook_task.py

```diff
@@ -21,87 +17,50 @@
 logger = logging.getLogger(__name__)
 
 
-# This needs to match the value defined in the Seer API:
-# https://github.com/getsentry/seer/blob/main/src/seer/routes/codegen.py
-SEER_PR_REVIEW_RERUN_PATH = "/v1/automation/codegen/pr-review/rerun"
-PREFIX = "seer.code_review.check_run.rerun"
+PREFIX = "seer.code_review.task"
 MAX_RETRIES = 3
 DELAY_BETWEEN_RETRIES = 60  # 1 minute
+RETRYABLE_ERRORS = (HTTPError,)
 
 
 @instrumented_task(
     name="sentry.seer.code_review.tasks.process_github_webhook_event",
     namespace=seer_code_review_tasks,
-    retry=Retry(times=MAX_RETRIES, delay=DELAY_BETWEEN_RETRIES, on=(HTTPError,)),
+    retry=Retry(times=MAX_RETRIES, delay=DELAY_BETWEEN_RETRIES, on=RETRYABLE_ERRORS),
     silo_mode=SiloMode.REGION,
 )
 def process_github_webhook_event(
-    *, organization_id: int, enqueued_at_str: str, **kwargs: Any
+    *, enqueued_at_str: str, organization_id: int | None = None, **kwargs: Any
```

Comment:
We're not really using `organization_id`. I may remove it in the next PR since changing the signature during deployment could affect task scheduling.